providers/org/datacite/normalizer.py

```diff
@@ -0,0 +1,585 @@
+import re
+import logging
+
+from share.normalize import ctx, tools
+from share.normalize.parsers import Parser
+from share.normalize.utils import format_doi_as_url
+
+
+logger = logging.getLogger(__name__)
+
+THE_REGEX = re.compile(r'(^the\s|\sthe\s)')
+DATE_REGEX = re.compile(r'(^[^a-zA-Z]+$)')
+
+
+def force_text(data):
```

Comment:
Do you know how this will deal with `None`s? There's another version in the oai harvester that blew up in the face of `None`.

Here's the offending tag:
```xml
<dc:title/>
```